helm/dex-app/templates/dex/secret.yaml

```diff
@@ -80,11 +84,7 @@ stringData:
       {{- end }}
       name: dex-k8s-authenticator
       secret: {{ .Values.oidc.staticClients.dexK8SAuthenticator.clientSecret }}
-      {{- if (or .Values.oidc.staticClients.gsCLIAuth.clientID .Values.oidc.staticClients.happa.clientID) }}
-      trustedPeers:
-      - {{ .Values.oidc.staticClients.gsCLIAuth.clientID }}
-      - {{ .Values.oidc.staticClients.happa.clientID }}
-      {{- end }}
+      {{- include "trusted-peers" (dict "trustedPeers" .Values.oidc.staticClients.dexK8SAuthenticator.trustedPeers "staticTrustedPeers" (list .Values.oidc.staticClients.happa.clientID .Values.oidc.staticClients.gsCLIAuth.clientID) ) }}
```

Comment:
Does it need to be a `dict`? Cant you use `concat` here?

Reply:
It would work if `trustedPeers` of all static clients in the default Values were set to empty array.
Otherwise concat fails in case the `trustedPeers` do not exist.

Reply:
Maybe `default` template function could help. If no let’s proceed as is

Reply:
Yes, that helped. Applied the change.